core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1180,6 +1180,14 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val IO_LOAD_LARGE_FILE_WARNING_THRESHOLD =
+    ConfigBuilder("spark.io.loadLargeFile.warning.threshold")
```

Comment:
The config name needs to represent a reasonable namespace hierarchy, which seems not the case here. how about `spark.io.warning.largeFileThreshold`?